sdk/src/main/scala/io/horizen/account/api/http/route/AccountTransactionApiRoute.scala

```diff
@@ -666,66 +666,72 @@ case class AccountTransactionApiRoute(override val settings: RESTApiSettings,
 
     val accountState = sidechainNodeView.getNodeState
     val epochNumber = accountState.getConsensusEpochNumber.getOrElse(0)
-
-    if (!Version1_4_0Fork.get(epochNumber).active) {
+    val fork14 = Version1_4_0Fork.get(epochNumber)
+    if (!fork14.active) {
       ApiResponseUtil.toResponse(GenericTransactionError(s"Fork 1.4 is not active, can not invoke this command",
         JOptional.empty()))
     }
     else if (!accountState.forgerStakesV2IsActive) {
       ApiResponseUtil.toResponse(GenericTransactionError(s"Forger Stake Storage V2 is not active, can not invoke this command",
         JOptional.empty()))
+    }
+    else if (operation == ForgerStakeV2MsgProcessor.UpdateForgerCmd &&
+      epochNumber < (fork14.mainnetActivationEpoch + NUM_OF_EPOCHS_AFTER_FORK_ACTIVATION_FOR_UPDATE_FORGER) ) {
```

Comment:
Shouldn't be selected the activation epoch for the correct network, i.e for mainnet mainnetActivationEpoch, for regstest regtestActivationEpoch and for testnet testnetActivationEpoch? They are not necessarily the same

Reply:
As discussed, the approach suggested by Paolo allow us to overcome also this issue.